pkg/controller/clusterclaim/clusterclaim_controller.go

```diff
@@ -35,22 +38,48 @@ const (
 // Add creates a new ClusterClaim Controller and adds it to the Manager with default RBAC. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager) error {
-	return AddToManager(mgr, NewReconciler(mgr))
+	logger := log.WithField("controller", ControllerName)
+	hiveControllersConfig := &corev1.ConfigMap{}
+	if err := mgr.GetAPIReader().Get(context.TODO(), types.NamespacedName{Namespace: controllerutils.GetHiveNamespace(), Name: hive.HiveControllersConfigMapName}, hiveControllersConfig); err != nil {
```

Comment:
Had never seen this GetAPIReader before, looked it up and the comments indicate:

```
	// GetAPIReader returns a reader that will be configured to use the API server.
	// This should be used sparingly and only when the client does not fit your
	// use case.
```

Is there a reason we need this here vs just using the mgr.GetClient? Is it not initialized yet? If so we should add a comment.

Was mounting the configmap into the pod considered?

Reply:
My bad I thought GetAPIReader is the only way to interact with the API server. Using GetClient now.